tests/layer_tests/onnx_tests/README.md

```diff
@@ -0,0 +1,135 @@
+#ONNX layer test coverage
```

Comment:
@lazarevevgeny @GlebKazantaev should we add the file? Is it maintained by someone?

Reply:
Let's remove this file.